draft-ietf-rtcweb-jsep.xml

```diff
@@ -1609,6 +1609,30 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             section below.
           </t>
         </section>
+        <section title="direction" anchor="sec.transceiver-direction">
+          <t>
+            The direction method returns the last value passed into
+            setDirection. If setDirection has never been called, it
+            returns the direction the transceiver was initialized with.
+          </t>
+        </section>
+        <section title="currentDirection" anchor="sec.transceiver-current-direction">
+          <t>
+            The currentDirection method returns the last negotiated
+            direction for the transceiver's associated m= section. More
+            specifically, it returns the <xref target="RFC3264"></xref>
+            directional attribute of the associated m= section in the last
+            applied answer, with "send" and "recv" directions reversed if
+            it was a remote answer. For example, if the directional
+            attribute for the associated m= section in a remote answer is
+            "recvonly", currentDirection returns "sendonly".
+          </t>
+          <t>
+            If an answer has never been applied containing an m= section
```

Comment:
awkward - suggest 
"If the transceiver has no associated m= section, perhaps because no answer that references this m= section has been applied, or if the transceiver is stopped..."